src/common/src/common.cpp

```diff
@@ -21,18 +21,24 @@
 // |  DEALINGS IN THE SOFTWARE.                                                  |
```

Comment:
I think the diff on this file got screwed up and confused version.hpp with common.cpp

Reply:
Yeah, I can't do much about that, unfortunately.